src/main/webapp/WEB-INF/web.xml

```diff
@@ -51,24 +51,36 @@
     <url-pattern>/register</url-pattern>
   </servlet-mapping>
 
-		<servlet>
-			<servlet-name>UserProfileServlet</servlet-name>
-			<servlet-class>codeu.controller.UserProfileServlet</servlet-class>
-		</servlet>
 
-		<servlet-mapping>
-			<servlet-name>UserProfileServlet</servlet-name>
-			<url-pattern>/userprofile/*</url-pattern>
-		</servlet-mapping>
+  <servlet>
+    <servlet-name>AdminServlet</servlet-name>
+    <servlet-class>codeu.controller.AdminServlet</servlet-class>
+  </servlet>
+
+  <servlet-mapping>
+    <servlet-name>AdminServlet</servlet-name>
+    <url-pattern>/admin</url-pattern>
+  </servlet-mapping>
+
+  <servlet>
+    <servlet-name>LogoutServlet</servlet-name>
+    <servlet-class>codeu.controller.LogoutServlet</servlet-class>
+  </servlet>
+
+  <servlet-mapping>
+    <servlet-name>LogoutServlet</servlet-name>
+    <url-pattern>/logout</url-pattern>
+  </servlet-mapping>
+
 
-		<servlet>
-			<servlet-name>AdminServlet</servlet-name>
-			<servlet-class>codeu.controller.AdminServlet</servlet-class>
-		</servlet>
-		<servlet-mapping>
-			<servlet-name>AdminServlet</servlet-name>
-			<url-pattern>/admin</url-pattern>
-		</servlet-mapping>
+<servlet>
```

Comment:
If you want to be consistent with the rest of the file, you should use spaces here as well.  You probably have to change your XML formatting or something in Eclipse.